egs2/qasr_tts/tts1/local/data_download.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Massa Baali
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+download_dir=$1
+
+# check arguments
+if [ $# != 1 ]; then
+    echo "Usage: $0 <download_dir>"
+    exit 1
+fi
+
+set -euo pipefail
+echo $download_dir
+cwd=$(pwd)
+
+if [ ! -d "${download_dir}/qasr_tts-1.0" ] && [ -f "qasr_tts-1.0.zip" ]; then
+    mkdir -p "${download_dir}"
+    cd "${download_dir}"
+    unzip "${cwd}/qasr_tts-1.0"
+    cd "${cwd}"
+    echo "successfully prepared data."
+    
+else
+    echo "Go to this link https://arabicspeech.org/qasr_tts "
```

Comment:
The link is not working.

Reply:
I will activate it once the code is published.

Reply:
Sounds cool!